cg/models/flow_cell/flow_cell.py

```diff
@@ -249,7 +249,7 @@ def is_flow_cell_ready(self) -> bool:
             LOG.warning(f"Copy of sequence data is not ready for flow cell {self.id}")
             return False
         LOG.debug(f"All data has been transferred for flow cell {self.id}")
-        LOG.info(f"Flow cell {self.id} is ready for downstream processing")
+        LOG.info(f"Flow cell {self.id} is ready for demultiplexing or encrypting")
```

Comment:
I would keep downstream processing. Because what about backup, cleaning etc.?

Reply:
Ok I will change it 👍